src/main/app/controller/courts/EditCourtGeneralController.ts

```diff
@@ -2,54 +2,18 @@ import {Response} from 'express';
 import autobind from 'autobind-decorator';
 import {AuthedRequest} from '../../../types/AuthedRequest';
 import {CourtPageData} from '../../../types/CourtPageData';
-import {isObjectEmpty} from '../../../utils/validation';
 
 @autobind
 export class EditCourtGeneralController {
   public async get(req: AuthedRequest, res: Response): Promise<void> {
     const updated = req.query.updated === 'true';
     const pageData: CourtPageData = {
       isSuperAdmin: req.session.user.isSuperAdmin,
-      court: {},
+      court: null,
```

Comment:
Is defaulting to null going to cause any issue? It looks like it gets set on line 16, so it isn't a huge problem if not. Is there a way to set a default empty value instead? 

Worried about null pointer exceptions, but this may not be a concern on the frontend, your call!

Reply:
I've updated the code to remove the requirement for null.